pkg/collector/corechecks/checkbase.go

```diff
@@ -48,6 +54,39 @@ func (c *CheckBase) BuildID(instance, initConfig integration.Data) {
 	c.checkID = check.BuildID(c.checkName, instance, initConfig)
 }
 
+// Configure is provided for checks that require no config. If overriden,
+// the call to CommonConfigure must be preserved.
+func (c *CheckBase) Configure(data integration.Data, initConfig integration.Data) error {
+	return c.CommonConfigure(initConfig)
+}
+
+// CommonConfigure is called when checks implement their own Configure method,
+// in order to setup common options (run interval, empty hostname)
+func (c *CheckBase) CommonConfigure(initConfig integration.Data) error {
+	commonOptions := integration.CommonInitConfig{}
+	err := yaml.Unmarshal(initConfig, &commonOptions)
+	if err != nil {
+		log.Errorf("error in yaml %s", err)
```

Comment:
Could we add more context to ease the debug when looking at flares ?

Reply:
Tweaked the strings.
BTW, is that OK to fail here, while we might have not before? As a user, I'd rather have the fail check if I fat-finger my conf instead of just trowing a warning and contining, but could this break upgrades?